contracts/MarketOracle.sol

```diff
@@ -5,33 +5,58 @@ import "./IOracle.sol";
 import "./SignatureLib.sol";
 
 contract MarketOracle is IOracle {
-
-    mapping (bytes32 => bytes32) private _results;
-    address immutable private _owner;
+    mapping(bytes32 => bytes32) private _results;
+    address private immutable _owner;
 
     constructor() {
         _owner = msg.sender;
     }
 
-    function checkResult(bytes32 marketId, bytes32 propositionId) external view returns (bool) {
-        require(propositionId != 0x0000000000000000000000000000000000000000000000000000000000000000, "getBinaryResult: Invalid propositionId");
+    function checkResult(
+        bytes32 marketId,
+        bytes32 propositionId
+    ) external view returns (bool) {
+        require(
+            propositionId !=
+                0x0000000000000000000000000000000000000000000000000000000000000000,
```

Comment:
I am happy to keep  what is here, just thought i would mention that I think you could use ethers.constants.HashZero for something like this in the future

Reply:
Wouldn't that be in the tests?

Reply:
This is old code, not sure why it has come up as a change

Reply:
Can't use ethers in solidity though @OptimusOpus

Reply:
Oh whoops, my bad